server/protocols/webadmin-integration-test/src/test/java/org/apache/james/webadmin/integration/WebAdminServerIntegrationTest.java

```diff
@@ -309,6 +309,22 @@ public void addressForwardsEndpointShouldListForwardAddresses() throws Exception
         assertThat(members).containsOnly("from1@domain.com", "from2@domain.com");
     }
 
+    @Test
+    public void addressAliasesEndpointShouldListAliasesAddresses() throws Exception {
+        dataProbe.addAliasMapping("alias1", "domain.com", "to1@domain.com");
+        dataProbe.addAliasMapping("alias2", "domain.com", "to2@domain.com");
```

Comment:
No I prefer a PUT.

We should try hard to keep our test suite as isolated from the implementation as possible,  this ensures that we can change the implementation whenever we want without having the test suite blocking our way.

This `DataProbe` craziness is just additional junk wiring that do not add value compared to the webadmin API.

Advanced, strongly typed RRT support by the CLI is not planned anytime soon (nor desirable to promote webadmin adoption) hence I would advocate not using it here, not creating methods in DataProbe for Alias management, and rather rely on webadmin.

For the sake of consistency, I would value using webadmin to replace `addForwardMapping` `removeForwardMapping` `addGroupMapping` `removeGroupMapping` in DataProbe and rather use webadmin.